ui/src/shared/reducers/annotations.ts

```diff
@@ -1,24 +1,44 @@
-import {ADDING, EDITING, TEMP_ANNOTATION} from 'src/shared/annotations/helpers'
+import {
+  ADDING,
+  EDITING,
+  DEFAULT_ANNOTATION,
+} from 'src/shared/annotations/helpers'
 
-import {Action} from 'src/types/actions/annotations'
-import {AnnotationInterface} from 'src/types'
+import {Action} from 'src/shared/actions/annotations'
+import {Annotation, TagFilter} from 'src/types/annotations'
 
 export interface AnnotationState {
+  annotations: {
+    [annotationId: string]: Annotation
```

Comment:
personally i'd prefer `ID` here instead of `Id` for consistency, unless you're up for changing all instances of ID to Id. the concern is less pronounced now that we're using typescript, but there are still components not in TS, and this has been a subtle source of bugs in the past. thoughts?